step.yml

```diff
@@ -67,19 +67,26 @@ inputs:
         - "yes"
         - "no"
       is_required: true
-  - additional_params: 
+  - additional_params:
     opts:
       title: "Additional parameters"
       summary: "The flags from this input field will be appended to the `flutter test` command."
       description: "The flags from this input field will be appended to the `flutter test` command."
+  - tests_path_pattern:
+    opts:
+      title: "Test files pattern"
+      summary: "This pattern will be used as the last argument appended to the `flutter test` command."
+      description: |-
+        The pattern from this input field will be expanded and fed to the `flutter test` command.  
+        Both * and ** glob patterns are supported. E.g.: `lib/**/*_test.dart`
```

Comment:
Both `*` and `**` glob patterns are supported. For example, `lib/**/*_test.dart`.